Microsoft.Research/Contracts/System/System.Net.WebRequest.cs

```diff
@@ -190,7 +200,18 @@ public static IWebProxy DefaultWebProxy
     //   System.NotImplementedException:
     //     If the property is not overridden in a descendant class, any attempt is made
     //     to get or set the property.
-    extern public virtual string Method { get; set; }
+    public virtual string Method
+    {
+      get
+      {
+        Contract.Ensures(!String.IsNullOrEmpty(Contract.Result<string>()));
```

Comment:
:bulb: I'll be pushing for uniformly referring to built-in types using their built-in names, which is super easy to explain to users and also super easy to automate. This comment is informational; *if* I'm successful it would be incorporated as part of #63 or a subsequent PR (e.g. to enable a subset of StyleCopAnalyzers).